### ISSUE TITLE
Implement `default_language_version` mapping

### ISSUE THREAD

**Issue by asottile** [pre-commit/pre-commit#647]
This would allow you to blanket choose an interpreter version across all hooks implemented in that language.

This is described in some detail [here](https://github.com/pre-commit/pre-commit/issues/644#issuecomment-339770094) though the rough idea would look something like:

```yaml
default_language_version:
    python: python2.7
    ruby: 2.1.5
```

This would give precedence to manually configured (either through `.pre-commit-hooks.yaml` or `.pre-commit-config.yaml`) versions and only overwrite in the case where `default` is not specified.  (precedence: manually specified > default_language_version > `language.get_default_version()`)

**Comment by asottile** [comment]
#908 implements this

**Comment by asottile** [comment]
This has been released as part of [v1.14.0](https://github.com/pre-commit/pre-commit/releases/tag/v1.14.0) 🎉!
